ph5/core/ph5api.py

```diff
@@ -1283,7 +1283,11 @@ def cut(self, das, start_fepoch, stop_fepoch, chan=1,
                     das_t.append(d)
         # adjust the number of data samples as to not over extend the
         # cut_stop_fepoch
-        calc_stop_fepoch = trace_start_fepoch + (len(data) / sr)
+        if data is None:
+            return [Trace(np.array([]), start_fepoch, 0., 0, sample_rate,
+                          None, None, das_t, None, None, clock=clock)]
+        else:
```

Comment:
I don't think this else block is necessary the above if returns.

Reply:
This seems to be resolved. Is there anything else that I should do so the requested changes can be closed?